webview-ui/src/components/chat/ChatTextArea.tsx

```diff
@@ -1443,6 +1446,43 @@ const ChatTextArea = forwardRef<HTMLTextAreaElement, ChatTextAreaProps>(
 						// Drag-over styles moved to DynamicTextArea
 						transition: "background-color 0.1s ease-in-out, border 0.1s ease-in-out",
 					}}>
+					<div
+						style={{
+							position: "absolute",
+							inset: "10px 15px", // match textarea/hightlight inset
+							pointerEvents: "none",
+							zIndex: 1, // Above textarea but below menus
+							overflow: "hidden", // clip shader to rounded rect
+							borderRadius: 2, // match textarea radius
+							transition: "opacity 1s ease-in-out",
+							opacity: isVoiceRecording ? 1 : 0,
+						}}>
```

Comment:
Can't we move these to the div above? Or remove the empty div at the first level?
```suggestion
					<div
						style={{
							position: "absolute",
							inset: "10px 15px", // match textarea/hightlight inset
							pointerEvents: "none",
							zIndex: 1, // Above textarea but below menus
							overflow: "hidden", // clip shader to rounded rect
							borderRadius: 2, // match textarea radius
							transition: "opacity 1s ease-in-out",
							opacity: isVoiceRecording ? 1 : 0,
						}}>
```